src/MeetingsSDKAdapter.js

```diff
@@ -94,6 +97,12 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
           localVideo: stream.getVideoTracks()[0],
         };
         break;
+      case MEDIA_TYPE_REMOTE_AUDIO:
+        this.meetings[ID] = {...meeting, remoteAudio: stream};
+        break;
+      case MEDIA_TYPE_REMOTE_VIDEO:
+        this.meetings[ID] = {...meeting, remoteVideo: stream};
+        break;
       default:
         this.meetings[ID] = {...meeting};
```

Comment:
Is there a reason why we are replacing the existing meeting object with a new object copy if nothing has changed?